[BUG] Latest release shipping broken

### Describe the bug

.

### Steps to reproduce

When I run the latest release it crashes instantly.

### Expected behavior

.

### Screenshots

_No response_

### Device information

- Device: [e.g. Google Pixel 6]
- OS: [e.g. stock ROM Android version 12.0]
- App version: [e.g. 12.1.0 Alpha 4]
.

### Additional context

_No response_

I confirm this. Had to revert to the second last version to stop the permanent crashes.

The log:

[Device Brand]: google
[Device Model]: Pixel 7
[Android Version]: 15
[Android API Level]: 35
[Process ID]: 8792
[User ID]: 0
[CPU ABI]: arm64-v8a
[Package Name]: app.lawnchair
[Version Name]: 15.0.0 (527f90d)
[Version Code]: 15000000
[Target SDK]: 35
[Min SDK]: 26
[Error Type]: JVM
[Crash Time]: 2025-05-08T06:34:43.903
[Stack Trace]:
java.lang.RuntimeException: Unable to start activity ComponentInfo{app.lawnchair/app.lawnchair.LawnchairLauncher}: java.lang.RuntimeException: Field preferences_ for t4.e not found. Known fields are [public v4.f0 t4.e.e, public static final t4.e t4.e.f, public static volatile v4.s t4.e.g]
	at android.app.ActivityThread.performLaunchActivity(ActivityThread.java:4206)
	at android.app.ActivityThread.handleLaunchActivity(ActivityThread.java:4393)
	at android.app.servertransaction.LaunchActivityItem.execute(LaunchActivityItem.java:222)
	at android.app.servertransaction.TransactionExecutor.executeNonLifecycleItem(TransactionExecutor.java:133)
	at android.app.servertransaction.TransactionExecutor.executeTransactionItems(TransactionExecutor.java:103)
	at android.app.servertransaction.TransactionExecutor.execute(TransactionExecutor.java:80)
	at android.app.ActivityThread$H.handleMessage(ActivityThread.java:2773)
	at android.os.Handler.dispatchMessage(Handler.java:109)
	at android.os.Looper.loopOnce(Looper.java:232)
	at android.os.Looper.loop(Looper.java:317)
	at android.app.ActivityThread.main(ActivityThread.java:8934)
	at java.lang.reflect.Method.invoke(Native Method)
	at com.android.internal.os.RuntimeInit$MethodAndArgsCaller.run(RuntimeInit.java:591)
	at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:911)
Caused by: java.lang.RuntimeException: Field preferences_ for t4.e not found. Known fields are [public v4.f0 t4.e.e, public static final t4.e t4.e.f, public static volatile v4.s t4.e.g]
	at v4.k0.G(Unknown Source:62)
	at v4.k0.x(Unknown Source:677)
	at v4.p0.a(Unknown Source:138)
	at t4.e.j(Unknown Source:24)
	at u4.g.a(Unknown Source:0)
	at q4.o0.a(Unknown Source:97)
	at q4.q.invokeSuspend(Unknown Source:38)
	at q4.q.invoke(Unknown Source:24)
	at q4.t0.a(Unknown Source:92)
	at q4.m0.i(Unknown Source:15)
	at q4.m0.e(Unknown Source:224)
	at q4.l.invokeSuspend(Unknown Source:127)
	at q4.l.invoke(Unknown Source:10)
	at q4.e1.b(Unknown Source:93)
	at e2.d.g(Unknown Source:92)
	at e2.d.y(Unknown Source:120)
	at q4.m0.h(Unknown Source:95)
	at q4.f0.invokeSuspend(Unknown Source:54)
	at wd.a.resumeWith(Unknown Source:8)
	at pe.g0.run(Unknown Source:114)
	at bg.x.run(Unknown Source:104)
	at we.j.run(Unknown Source:2)
	at we.a.run(Unknown Source:93)

--- a/build.gradle
+++ b/build.gradle
@@ -418,7 +418,7 @@ dependencies {
     implementation "com.patrykmichalik.opto:domain:$optoVersion"
     implementation "com.patrykmichalik.opto:core:$optoVersion"
     implementation "com.patrykmichalik.opto:compose:$optoVersion"
-    implementation "androidx.datastore:datastore-preferences:1.1.6"
+    implementation "androidx.datastore:datastore-preferences:1.1.4"
     def retrofitVersion = "2.11.0"
     implementation "com.squareup.retrofit2:retrofit:$retrofitVersion"
     implementation "com.squareup.retrofit2:converter-kotlinx-serialization:$retrofitVersion"

